convert/convert/converters/labelstudio/labelstudio_format.py

```diff
@@ -0,0 +1,350 @@
+from pathlib import Path
+from typing import Any, Dict, List, Optional
+
+import requests
+from fastapi import HTTPException, status
+
+from convert.config import settings
+from convert.converters.base_format.models.annotation import (
+    BadgerdocAnnotation,
+)
+from convert.converters.base_format.models.manifest import Manifest
+from convert.converters.base_format.models.tokens import BadgerdocToken, Page
+from convert.logger import get_logger
+
+from .models.annotation import (
+    Annotation,
+    Data,
+    DocumentRelation,
+    LabelStudioModel,
+    Meta,
+    ModelItem,
+    ResultItem,
+    Value,
+)
+
+LOGGER = get_logger(__file__)
+LOGGER.setLevel("DEBUG")
+
+
+class LabelStudioFormat:
+    DEFAULT_ID_FOR_ONE_ANNOTATION = 1
+
+    # TODO: support more than 1 page
+    def __init__(self) -> None:
+        self.labelstudio_data = LabelStudioModel()
+
+    @staticmethod
+    def form_token_text(token: BadgerdocToken) -> str:
+        text = f"{token.previous or ''}{token.text}{token.after or ''}"
+        return text
+
+    def from_badgerdoc(
+        self,
+        badgerdoc_tokens: Page,
+        badgerdoc_annotations: Optional[BadgerdocAnnotation],
+        badgerdoc_manifest: Optional[Manifest],
+        request_headers: Dict[str, str],
+    ):
+        text = "".join(
+            [self.form_token_text(obj) for obj in badgerdoc_tokens.objs]
+        )
+
+        annotations = []
+        if badgerdoc_annotations:
+            objs = self.convert_annotation_from_bd(
+                badgerdoc_annotations, badgerdoc_tokens.objs, text
+            )
+            relations = self.convert_relation_from_bd(badgerdoc_annotations)
+            annotations.append(
+                Annotation(
+                    id=self.DEFAULT_ID_FOR_ONE_ANNOTATION,
+                    result=objs + relations,
+                )
+            )
+
+        meta = Meta()
+        if badgerdoc_manifest:
+            meta = self.form_meta(badgerdoc_manifest, request_headers)
+
```

Comment:
It looks weird a bit for me.
if badgerdoc_manifest, then we create "meta" and rewrite it in the next step.

maybe this way:
self.form_meta(badgerdoc_manifest, request_headers) if badgerdoc_manifest else Meta()

or:
if badgerdoc_manifest:
.....
else:
......